dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/trackedentitycomment/hibernate/TrackedEntityComment.hbm.xml

```diff
@@ -11,7 +10,6 @@
     <id name="id" column="trackedentitycommentid">
       <generator class="native" />
     </id>
-    &identifiableProperties;

```

Comment:
We need to add the mappings directly here so that Hibernate queries will still work, just without the not-null constraints:

    <property name="uid" column="uid" unique="true" length="11" not-null="false" />
    <property name="code" column="code" not-null="false" unique="true" length="50" />
    <property name="created" type="timestamp" not-null="false" />
    <property name="lastUpdated" type="timestamp" not-null="false"/>
    <many-to-one name="lastUpdatedBy" class="org.hisp.dhis.user.User" column="lastupdatedby" foreign-key="fk_lastupdateby_userid" />

Reply:
True. Somehow forgot about it that this says Hibernate how it should behave ... Fixed.